sound/soc/sof/sof-audio.h

```diff
@@ -88,6 +90,9 @@ struct ipc_tplg_control_ops {
 	void (*update)(struct snd_sof_dev *sdev, void *ipc_control_message);
 	/* Optional callback to setup kcontrols associated with an swidget */
 	int (*widget_kcontrol_setup)(struct snd_sof_dev *sdev, struct snd_sof_widget *swidget);
+	/* callback to set up volume table for volume kcontrols */
```

Comment:
do we need to say explicitly it's mandatory callback?

Reply:
I was torn about it. If we have a volume kcontrol in topology it is mandatory but not otherwise. So when parsing topology, we return an error if the op is not set.

Reply:
that tells me it's mandatory, we cannot control when people enable volume control.

adding a comment 'mandatory when volume control is enabled' would be fine as well.

Reply:
ok fixed now. Thanks!